code/game/machinery/computer/computer.dm

```diff
@@ -134,6 +134,9 @@
 		if(ishuman(usr))
 			playsound(src, pick('sound/machines/computer_typing4.ogg', 'sound/machines/computer_typing5.ogg', 'sound/machines/computer_typing6.ogg'), 5, 1)
 
+/obj/structure/machinery/computer/get_applying_acid_time()
+	return -1
+
```

Comment:
imo little reason to keep this restriction

Reply:
Removed this.